common/src/dev-container-cli.ts

```diff
@@ -25,18 +23,16 @@ function getSpecCliInfo() {
   };
 }
 
-async function isCliInstalled(exec: ExecFunction): Promise<boolean> {
+async function isCliInstalled(exec: ExecFunction, cliVersion: string): Promise<boolean> {
   try {
-    const {exitCode} = await exec(getSpecCliInfo().command, ['--help'], {
-      silent: true,
-    });
-    return exitCode === 0;
+    const {exitCode, stdout} = await exec(getSpecCliInfo().command, ['--version'], {});
+    return exitCode === 0 && stdout === cliVersion;
```

Comment:
Could stdout include a newline? Maybe use `.trim()` on the string.

Reply:
Good point, I have addressed the change.